src/python/WMCore/MicroService/Unified/MSRuleCleaner.py

```diff
@@ -60,9 +64,14 @@ def __init__(self, msConfig, logger=None):
         self.msConfig.setdefault("rucioWmaAccount", "wma_test")
         self.msConfig.setdefault("rucioMStrAccount", "wmcore_transferor")
         self.msConfig.setdefault('enableRealMode', False)
+
         self.mode = "RealMode" if self.msConfig['enableRealMode'] else "DryRunMode"
         self.emailAlert = EmailAlert(self.msConfig)
         self.curlMgr = RequestHandler()
+        self.wmstatsSvc = WMStatsServer(self.msConfig["wmstatsUrl"], logger=self.logger)
+        self.logDB = self.logDB = LogDB(self.msConfig["logDBUrl"],
```

Comment:
buggy :)

Reply:
@amaltaro excuse my ignorance here ;), but please do elaborate!

Reply:
repeated assignment

Reply:
Aaaand... this is a silly typo!!!  Fixing it now.
Thanks!